mod/contacts.php

```diff
@@ -595,7 +595,7 @@ function contacts_content(App $a, $update = 0)
 		/// @todo Only show the following link with DFRN when the remote version supports it
 		$follow = '';
 		$follow_text = '';
-		if (in_array($contact['network'], [Protocol::DIASPORA, Protocol::OSTATUS, Protocol::DFRN])) {
+		if ($contact['network'] != Protocol::STATUSNET) {
```

Comment:
This "native support" constant needs to be added here as well. It's too late for a full review, I will have a look at it in the morning.